.circleci/config.yml

```diff
@@ -105,9 +105,15 @@ jobs:
             cd src/api
             pickfile=log/pick.$CIRCLE_NODE_INDEX.list
 
-            # single out db specs
-            circleci tests glob 'spec/**/*_spec.rb' | grep -v 'spec/db/' > spec.list
-            circleci tests split --total $CIRCLE_NODE_TOTAL --split-by=timings < spec.list > $pickfile
+            # reserve the latest node for beta
```

Comment:
It is also for `db` specs as stated on line 113.

Reply:
In this `if` statement we use the last container to run only the beta tests. In the `else` section we exclude both, `beta` and `db` specs.

The `db` specs are run in container 0, see line 122.